standards/application/waku-api.md

```diff
@@ -107,90 +107,152 @@ language_mappings:
     module_type: "esm"
 ```
 
-
 ### Type definitions
 
 ```yaml
 types:
   Config:
     type: struct
     fields:
-      operating_mode: 
+      mode:
         type: string
-        constraints: ["edge", "relay"]
-        description: "The mode of operation of the Waku node. Core protocols used by the node are inferred from this mode."
+        constraints: ["client", "server"]
+        description: "The mode of operation of the Waku node.
+                      Waku core protocols used by the node are inferred from this mode."
       network_config:
         type: struct
-        default: TheWakuNetworkPreset
         fields:
-          boostrap_nodes:
+          boostrap_servers:
             type: array<string>
-            default: ""
-            description: "Bootstrap nodes, entree and multiaddr formats are accepted."
-          static_store_nodes:
+            default: "enrtree://AIRVQ5DDA4FFWLRBCHJWUWOO6X6S4ZTZ5B667LQ6AJU6PEYDLRD5O@sandbox.waku.nodes.status.im"
```

Comment:
I disagree with this approach because these are not independent default values, they are default values for TWN.

Hence, I propose to continue with the original approach to define a `TheWakuNetwork` hardcoded value`. See original PR.